### ISSUE TITLE
TypeError: the JSON object must be str, bytes or bytearray, not NoneType

### ISSUE THREAD

**Issue by roksys** [Kozea/wdb#127]
Hey,

I am having trouble to start wdb. I have wdb and wdb.server installed in my virtualenv.

I am placing `import wdb; wdb.set_trace()` inside my code, trying to start  wdb server by running `wdb.server.py &` and trying to run my code with placed debug point.

```
$ wdb.server.py &
[4] 51913
Traceback (most recent call last):
  File "/Users/rokas/.virtualenvs/virtualenv/bin/wdb.server.py", line 10, in <module>
    from wdb_server import server
  File "/Users/rokas/.virtualenvs/virtualenv/lib/python3.7/site-packages/wdb_server/__init__.py", line 374, in <module>
    raise_error=False
TypeError: fetch() got multiple values for argument 'raise_error'
[4]   Exit 1                  wdb.server.py
```

```
$ python --version
Python 3.7.2
$ pip show wdb
Name: wdb
Version: 3.2.5
$ pip show wdb.server
Name: wdb.server
Version: 3.2.5
```

**Comment by blackmamo** [comment]
I too have a similar issue. Did you resolve this?

**Comment by blackmamo** [comment]
It seems that the issue here is that the tornado http client returns an  Awaitable["HTTPResponse"], and doesn't take a callback. The second argument is raise_error and should be a boolean. Hence the error. Commenting out the raise_error=False makes the debug server start up (but note that the callback is never invoked, but that doesn't seem criticial). I am not sure if this is a versioning issue and that earlier versions of tornado's http client had a different signature.

**Comment by buma** [comment]
I think the issue is Python 3.7. At least in README it is said that it works only up to Python 3.5. I'm not sure if README isn't updated or it really doesn't work in Python 3.7

I have same issue on same Python and wdb version. 
I "fixed" it by starting wdb.server in another Python 2 virtualenv. It starts the debugging portion but crashes because of Pygame used in my code.

**Comment by blackmamo** [comment]
The setup.py for wdb.server states tornado >= 5. In my environment this has resolved to tornado 6.0.1. The signature of fetch has changed between these two.

**Comment by buma** [comment]
You are correct. It also works here if I install tornado 5 and then install wdb.server on Python 3.
Still seems that Pygame and wdb don't work together for some reason. But that seems to be another bug.

**Comment by fessacchiotto** [comment]
Any solutions found on this yet? Thanks!

**Comment by paradoxxxzero** [comment]
Sorry for the long delay, I made it sync so it will work in any case now.

**Comment by okeyokoro** [comment]
I'm still hitting this error

**Comment by paradoxxxzero** [comment]
should be fixed in 3.2.6

**Comment by julianram** [comment]
Hi, 

The solution would be to install  pip3 install Tornado == 5.1 

Regards,
